apps/sms/js/utils.js

```diff
@@ -653,6 +653,7 @@
           canvas = Utils.imageToCanvas(
             img, adjustedSize.width, adjustedSize.height
           );
+          img.src = '';
```

Comment:
mmm, it looks like we do the same below in ```cleanup``` function that is called once this deferred resolved or rejected. Is there a reason why it can't wait for the ```cleanup``` function call?